src/deluge/dsp/timestretch/time_stretcher.cpp

```diff
@@ -950,8 +950,9 @@ bool TimeStretcher::hopEnd(SamplePlaybackGuide* guide, VoiceSample* voiceSample,
 
 #else
 	// If no one's reading from the buffer anymore, stop filling it
-	if (buffer && !olderHeadReadingFromBuffer) { // olderHeadReadingFromBuffer will always be false - we set it above,
-		                                         // at the start
+	if ((buffer != nullptr)
+	    && !olderHeadReadingFromBuffer) { // olderHeadReadingFromBuffer will always be false - we set it above,
```

Comment:
should !olderHeadReadingFromBuffer be more explicit like the other stuff?

Reply:
that's already a bool